hikari/impl/entity_factory.py

```diff
@@ -2572,6 +2627,13 @@ def deserialize_command_interaction(
 
         entitlements = [self.deserialize_entitlement(entitlement) for entitlement in payload.get("entitlements", ())]
 
+        authorizing_integration_owners = {
+            application_models.ApplicationIntegrationType(int(integration_type)): snowflakes.Snowflake(
+                int(integration_owner_id)
+            )
```

Comment:
I believe for snowflakes, you do not need to cast it to an int before hand (as Snowflake is a subclass of int)